### DISCUSSION TITLE
error* "Unable to process request due to missing initial state" when I'm trying to get token for my PC google assistant

### DISCUSSION THREAD

**Discussion by madmanswork** [Melvin-Abraham/Google-Assistant-Unofficial-Desktop-Client#872] [Q&A]
I get this error "Unable to process request due to missing initial state. This may happen if browser session Storage is inaccessible or accidentally cleared." when I'm using my email to get the token to connect my google assistant on PC. I've tried allowing every cookie when logging in, but it still doesn't work .Can someone help plz
![2023-02-27](https://user-images.githubusercontent.com/93680082/221750259-3daf2073-ace9-45e5-b4b3-fe7a638e46f1.png)
![2023-02-27 (1)](https://user-images.githubusercontent.com/93680082/221750267-8f4004a5-ac8e-4845-baaa-e91eb083ce2e.png)

**Comment by sumeshir26** [comment]
@madmanswork Copy the code from the address bar
